rswag-specs/lib/rswag/specs/openapi_formatter.rb

```diff
@@ -190,9 +181,16 @@ def parse_formdata_or_body_parameter(endpoint, parameter, mime_list)
           set_mime_config(mime_config, parameter)
           set_mime_examples(mime_config, endpoint)
           parse_parameter_required(endpoint, mime_config, parameter)
+
+          parameter[:schema][:description] = parameter[:description] if parameter[:description]
```

Comment:
This is the cause of difference from the current behavior.

Reply:
Thanks, yeah I can't remember why that line was in the code before. I moved it hence the change of behaviour probably, but I'll have a look at why it's even there in the first place. This code is all very hard to maintain btw, cause it relies on mutating the options that come from the spec DSL into the OpenAPI format and that mutating is hard to reason about, would be nice to eventually rewrite this so the openAPI output is generated from the ground up from the input options

Reply:
I agree it is a bit of maze to go through it.

I'm very happy for the refactoring done in this PR and take it as the first step in cleaning up the parsing in future.

Reply:
Ok hopefully now fixed!